packages/ui/src/working-groups/components/OpeningsList/OpeningsList.tsx

```diff
@@ -10,7 +10,7 @@ export interface OpeningsListProps {
   past?: boolean
 }
 
-export const OpeningsList = ({ openings, past }: OpeningsListProps) => {
+export const OpeningsList = ({ openings, past = true }: OpeningsListProps) => {
```

Comment:
agree, there's possible downside.
```suggestion
export const OpeningsList = ({ openings, past }: OpeningsListProps) => {
```